app/src/main/java/com/lagradost/cloudstream3/ui/player/AbstractPlayerFragment.kt

```diff
@@ -482,11 +485,13 @@ abstract class AbstractPlayerFragment(
         )
 
         if (player is CS3IPlayer) {
+            val ctx = context ?: return
             // preview bar
             val progressBar: PreviewTimeBar? = playerView?.findViewById(R.id.exo_progress)
             val previewImageView: ImageView? = playerView?.findViewById(R.id.previewImageView)
             val previewFrameLayout: FrameLayout? = playerView?.findViewById(R.id.previewFrameLayout)
-            if (progressBar != null && previewImageView != null && previewFrameLayout != null) {
+            val previewSeekThumb = PreferenceManager.getDefaultSharedPreferences(ctx)?.getBoolean(ctx.getString(R.string.preview_seekbar_key),false) == true
```

Comment:
This is somewhat incorrect, as what you are doing is simply hiding the UI. It will still be loaded in the background.